excalidraw-app/components/AppWelcomeScreen.tsx

```diff
@@ -55,17 +30,6 @@ export const AppWelcomeScreen: React.FC<{
               onSelect={() => props.onCollabDialogOpen()}
             />
           )}
-          {!isExcalidrawPlusSignedUser && import.meta.env.VITE_APP_PLUS_LP && (
```

Comment:
I would prefer leveraging these env variables to deactivate features, as this lessens the pain for future merges and eventually failing tests. I guess setting `import.meta.env.VITE_APP_PLUS_LP` to undefined in the config settings removes this as well?

Reply:
Flag is added.